QuizApp/Scenes/LoginPage/ViewController/LoginViewController.swift

```diff
@@ -9,15 +9,143 @@ import UIKit
+            .font: UIFont.systemFont(ofSize: FontSizes.xs)
+        ]
+        input.attributedPlaceholder = NSAttributedString(string: placeholderText, attributes: attributes)
+        input.layer.borderWidth = Sizing.inputBorder
+        input.layer.borderColor = Colors.border.cgColor
+        input.layer.cornerRadius = Sizing.inputRadius
+        return input
+    }()
     
+    private let startButton = YellowRoundedButton(
+        title: "ქვიზის დაწყება",
+        width: Sizing.startButtonWidth,
+        height: Sizing.startButtonHeight,
+        radius: Sizing.startButtoRadius,
+        fontSize: FontSizes.xs
+    )
     
+    // MARK: - Constraints
+    private var blueBackgroundTopConstraint: NSLayoutConstraint!
+    
+    // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .blue
+        setupUI()
+        setupKeyboardObservers()
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        removeKeyboardObservers()
+    }
+    
+    // MARK: - UI Setup
+    private func setupUI() {
+        setupView()
+        setupViewHierarchy()
+        setConstraints()
+    }
+    
+    private func setupView() {
+        view.backgroundColor = .white
+    }
+    
+    private func setupViewHierarchy() {
+        loginStackView.addArrangedSubviews(input, startButton)
+        view.addSubviews(blueBackground, titleLabel, illustration, loginStackView)
+        view.bringSubviewToFront(titleLabel)
+    }
+    
+    private func setConstraints() {
+        blueBackgroundTopConstraint = blueBackground.topAnchor.constraint(equalTo: view.topAnchor, constant: Sizing.blueBackgroundTopAnchor)
+        
+        NSLayoutConstraint.activate([
+            blueBackgroundTopConstraint,
+            blueBackground.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Sizing.blueBackgroundLeadingAnchor),
+            blueBackground.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: Sizing.blueBackgroundTrailingAnchor),
+            blueBackground.heightAnchor.constraint(equalToConstant: Sizing.blueBackgroundHeight),
+            
+            titleLabel.topAnchor.constraint(equalTo: blueBackground.topAnchor, constant: Sizing.labelTopAnchor),
+            titleLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            
+            illustration.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: Sizing.illustrationTopAnchor),
+            illustration.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            
+            loginStackView.topAnchor.constraint(equalTo: blueBackground.bottomAnchor, constant: Sizing.stackViewTopAnchor),
+            loginStackView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            
+            input.widthAnchor.constraint(equalToConstant: Sizing.inputWidth),
+            input.heightAnchor.constraint(equalToConstant: Sizing.inputHeight)
```

Comment:
when it is textfield signing widths is wrong approach, because for example textfield size is hardcoded as 100 it will look terrible for bigger or smaller devices you should apply  leading and trailing anchors to display it correctly on every device